src/main/java/org/cloudfoundry/promregator/endpoint/DiscoveryEndpoint.java

```diff
@@ -5,42 +5,35 @@
 
 import javax.servlet.http.HttpServletRequest;
 
+import org.cloudfoundry.promregator.config.PromregatorConfiguration;
 import org.cloudfoundry.promregator.discovery.CFMultiDiscoverer;
 import org.cloudfoundry.promregator.scanner.Instance;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Scope;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.context.WebApplicationContext;
 
 import com.fasterxml.jackson.annotation.JsonGetter;
 
 @RestController
 @RequestMapping(EndpointConstants.ENDPOINT_PATH_DISCOVERY)
-@Scope(value=WebApplicationContext.SCOPE_REQUEST)
+//@Scope(value=WebApplicationContext.SCOPE_REQUEST)
```

Comment:
If we don't need it anymore: get rid of it
(I am asking myself why it was there in first place - perhaps copy&paste?)

Reply:
Deleted. I'm guessing it was here due to copy/paste. The only reason I could think it would exist is for `public ResponseEntity<DiscoveryResponse[]> getDiscovery(HttpServletRequest request) {` so it would inject a fresh HttpServletRequest on each call.

However, that shouldn't be needed. So I've removed it